Subject: [PATCH] fix dependencies
---
Index: build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle.kts b/build.gradle.kts
--- a/build.gradle.kts	(revision dd230fedf552afb3aa6d530334e55c2fdcefd26a)
+++ b/build.gradle.kts	(revision 89e267148385a099a649cac555c46eb743e4424e)
@@ -11,7 +11,8 @@
     repositories {
         mavenCentral()
         mavenLocal()
-
+        google()
+        maven { url = uri("https://jitpack.io") }
         maven { url = uri("https://repo1.maven.org/maven2") }
         maven { url = uri("https://repo.spring.io/milestone") }
         maven { url = uri("https://repo.spring.io/snapshot") }
Index: demo-rabbit/build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/demo-rabbit/build.gradle.kts b/demo-rabbit/build.gradle.kts
--- a/demo-rabbit/build.gradle.kts	(revision dd230fedf552afb3aa6d530334e55c2fdcefd26a)
+++ b/demo-rabbit/build.gradle.kts	(revision 89e267148385a099a649cac555c46eb743e4424e)
@@ -2,11 +2,10 @@
 import org.springframework.boot.gradle.tasks.bundling.BootJar
 
 plugins {
-    id("org.springframework.boot") version "2.6.7"
-    id("io.spring.dependency-management") version "1.1.0"
+    id("org.springframework.boot")
+    id("io.spring.dependency-management")
     kotlin("jvm")
-    kotlin("plugin.spring") version "1.5.21"
-    id("nu.studer.jooq") version "6.0.1"
+    kotlin("plugin.spring")
 }
 java.sourceCompatibility = JavaVersion.VERSION_11
 group = "com.otus"
@@ -41,15 +40,13 @@
     implementation("com.arcadedb:arcadedb-engine:$arcadeDbVersion")
     implementation("com.arcadedb:arcadedb-network:$arcadeDbVersion")
     implementation("org.postgresql:postgresql:$postgresDriverVersion")
-    jooqGenerator("org.postgresql:postgresql:$postgresDriverVersion")
     //spring
     implementation("org.springframework.boot:spring-boot-starter")
     implementation("org.springframework.boot:spring-boot-starter-actuator")
     implementation("org.springframework.boot:spring-boot-starter-amqp")
-    implementation("org.springframework.boot:spring-boot-starter-jooq")
     implementation("org.springframework.boot:spring-boot-starter-web")
     //kotlin
-    implementation(kotlin("stdlib"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8") // for spring-boot app
     implementation("javax.xml.bind:jaxb-api:2.3.1")
     implementation("com.fasterxml.jackson.module:jackson-module-paranamer")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision dd230fedf552afb3aa6d530334e55c2fdcefd26a)
+++ b/gradle.properties	(revision 89e267148385a099a649cac555c46eb743e4424e)
@@ -1,8 +1,10 @@
 kotlin.code.style=official
 arcadeDbVersion=22.12.1
+kotlinVersion=1.7.20
 testContainersVersion=1.17.6
 postgresUrl=jdbc:postgresql://localhost:7450/demo_graph?currentSchema=graph
 postgresUser=demo
 postgresPassword=123456
 postgresDriverVersion=42.6.0
-kotlinVersion=1.7.0
\ No newline at end of file
+springframeworkBootVersion=2.7.3
+springDependencyManagementVersion=1.0.13.RELEASE
\ No newline at end of file
Index: settings.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle.kts b/settings.gradle.kts
--- a/settings.gradle.kts	(revision dd230fedf552afb3aa6d530334e55c2fdcefd26a)
+++ b/settings.gradle.kts	(revision 89e267148385a099a649cac555c46eb743e4424e)
@@ -2,7 +2,13 @@
 
 pluginManagement {
     plugins {
-        kotlin("jvm") version "1.5.21"
+        val kotlinVersion: String by settings
+        val springframeworkBootVersion: String by settings
+        val springDependencyManagementVersion: String by settings
+        kotlin("jvm") version kotlinVersion apply false
+        kotlin("plugin.spring") version kotlinVersion apply false
+        id("org.springframework.boot") version springframeworkBootVersion apply false
+        id("io.spring.dependency-management") version springDependencyManagementVersion apply false
     }
     repositories {
         mavenCentral()
